hydromt/models/plugins.py

```diff
@@ -13,7 +13,6 @@
 # local generic models
 LOCAL_EPS = {
     "vector_model": "hydromt.models.components.vector:VectorModel",
-    "mesh_model": "hydromt.models.components.mesh:MeshModel",
     "network_model": "hydromt.models.components.network:NetworkModel",
 }
```

Comment:
I can see this code is not up to date with `model-refactor`. if we're going to wait to get that in before you merge this you might want to merge it because I can already see it's going to cause a few merge issues here.